e2e/nomostest/git-server.go

```diff
@@ -35,6 +35,7 @@ const testGitServer = "test-git-server"
 const testGitServerImage = testing.TestInfraArtifactRegistry + "/git-server:v1.0.0"
 const testGitHTTPServer = "http-git-server"
 const testGitHTTPServerImage = testing.TestInfraArtifactRegistry + "/http-git-server:v1.0.0-b3b4984cd"
+const safeToEvictAnnotation = "cluster-autoscaler.kubernetes.io/safe-to-evict"
```

Comment:
This may not be safe to add to normal clusters, because they allow persistence for much much longer. It's only limited to 7 days on autopilot. So we probably want a check for that.

Reply:
Done